tox.ini

```diff
@@ -1,15 +1,18 @@
 [tox]
-envlist = py27
+envlist = tests, pylama
 
-[testenv:py27]
-deps =
-    mock
-    mozfile >= 1.1
-    mozprocess >= 0.22
-    pylama
-    pytest
-    wptserve >= 1.4.0
+[testenv:tests]
+usedevelop = true
+passenv = TRAVIS TRAVIS_JOB_ID TRAVIS_BRANCH
+deps = -rrequirements/tests.txt
```

Comment:
I don't understand. How do you make coverage conditional based on if it is Travis or local?
I only have one testenv and if I split it off we have the issue again that it runs the code twice, no?